libsplinter/src/rest_api/cors.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Provides CORS support for the REST API
+//!
+//! This is an experimental feature.  To enable, use the feature `"rest-api-cors"`.
```

Comment:
Should probably include a new line between this and the `use` statements. There's also an extra space in this line.